src/action/config/ActionConfiguration.cpp

```diff
@@ -25,9 +25,9 @@ ActionConfiguration::ActionConfiguration(
     const mesh::PtrMeshConfiguration &meshConfig)
     : NAME_DIVIDE_BY_AREA("divide-by-area"),
       NAME_MULTIPLY_BY_AREA("multiply-by-area"),
-      NAME_SCALE_BY_COMPUTED_DT_RATIO("scale-by-computed-dt-ratio"),
-      NAME_SCALE_BY_COMPUTED_DT_PART_RATIO("scale-by-computed-dt-part-ratio"),
-      NAME_SCALE_BY_DT("scale-by-dt"),
+      NAME_SCALING_BY_TIME_STEP_TO_TIME_WINDOW_RATIO("scale-by-computed-dt-ratio"),       //@todo rename this, breaking change!
+      NAME_SCALING_BY_COMPUTED_TIME_WINDOW_PART_RATIO("scale-by-computed-dt-part-ratio"), //@todo rename this, breaking change!
+      NAME_SCALING_BY_TIME_WINDOW_SIZE("scale-by-dt"),                                    //@todo rename this, breaking change!, currently misleading. See https://github.com/precice/precice/issues/934
```

Comment:
Could you please open an issue for milestone 3?
In this issue we should also discuss which actions to keep and which to drop completely.

Reply:
Done. https://github.com/precice/precice/issues/1009